external/fv3fit/fv3fit/tfdataset.py

```diff
@@ -27,11 +27,13 @@ def apply_to_mapping(
     return {name: tensor_func(tensor) for name, tensor in data.items()}
 
 
-@curry
 def apply_to_tuple(
```

Comment:
This was unused and broken before, I've added an external package test to confirm tuple entries are passed as *args to map functions.